### PR TITLE
user require syntax for partials

### PR DESCRIPTION
**Pull request by martell**
This is blocked by
https://github.com/parcel-bundler/parcel/issues/1744
which I now have a PR for.
Hopefully we can get this in before v1.9.8
I will update this PR to point to that tag then instead of using the master branch

### CODE REVIEW

**Comment by edm00se** [comment]
Thanks for the PR. I had seen parcel-bundler/parcel#1732 and was waiting for it to be included in a version release to make the changes. Once the npm version bumps and includes the update, I'll be happy to accept and merge this.

File: src/js/directives.js
Comment (edm00se) [inline src/js/directives.js]: One of the `, 'utf8'` second parameters got missed.

File: package.json
Comment (edm00se) [inline package.json]: I'd like to wait for the changes to be part of a version release. We'll need to update the version here once that's been done.

File: src/js/directives.js
Comment (edm00se) [inline src/js/directives.js]: One of the , 'utf8' second parameters got missed.

**Comment by edm00se** [comment]
Thanks for the PR. I had seen parcel-bundler/parcel#1732 and was waiting for it to be incorporated into a version release to do it myself. Once the npm version bumps and includes the update, I'll be happy to accept and merge this.

**Review by edm00se — CHANGES_REQUESTED** [review]
Thanks for the PR. I had seen parcel-bundler/parcel#1732 and was waiting for it to be included in a version release to make the changes. I am pretty excited for it. Once the npm version bumps and includes the update, I'll be happy to accept and merge this.

**Comment by martell** [comment]
I updated the PR to point to what I expect to be the next release version.

Thanks for working on this project to give people a good example on how to modernize their old angularjs 1.x apps.

**Review by edm00se — APPROVED** [review]

**Comment by absynce** [comment]
Should this update the [migration guide](https://github.com/edm00se/modernize-ng1/blob/master/docs/Migrate.md#updating-the-router) too? It currently shows an example of the prior way to load partials.

**Comment by edm00se** [comment]
@absynce Thanks for checking. Once merged I'll make sure to update the migration guide. You're right, it currently shows use of `fs.readFileSync`.

**Comment by edm00se** [comment]
Thanks again for the PR. It has been incorporated into #6 so it can get released tonight quickly while I had a few minutes. It was an easy way of handling the merge conflict that came up as well as upgrading another dep and adding the package-lock file. The app is now building with parcel 1.10.1! 🎉

I've also updated the `docs/Migrate.md` migration guide to reflect the direct use of `require`. 

For the wayward reader looking for context, the release of `parcel-bundler` after 1.9.7 wound up being `1.10.0`. Blog post: https://medium.com/@devongovett/parcel-v1-10-0-babel-7-flow-elm-and-more-c20736553573

**Comment by edm00se** [comment]
Support for IE is currently blocked until the html loader handling changes for either:

- the bundle loader not directly depending on `Promise` or `fetch` APIs in the browser (un-polyfilled, before any babel polyfill takes effect)
- the html partials not be separate bundles (appended to the existing bundle)

See #14.

**Pull request closed without merging**
